ajax/libs/require-jquery/package.json

```diff
@@ -13,5 +13,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/jrburke/require-jquery"
-  }
+  },
+  "license": "CC0-1.0"
```

Comment:
https://github.com/requirejs/require-jquery/blob/0.25.0/jquery-require-sample/webapp/scripts/require-jquery.js#L3
hmm... Need to confirm with the author first.

Reply:
@maruilian11  I think it is mistake of the author.
Here written black on white the license of the library
https://github.com/requirejs/require-jquery/blob/master/LICENSE

Reply:
Yes, Maybe it is. When we meet this situation, we usually confirm with the author for the politeness.

Reply:
@maruilian11 The issue was opened

Reply:
@PeterDaveHello 
@maruilian11 
We have to decide what our steps in this case... it can  be in a suspended state

Reply:
@PeterDaveHello What do you think? Can we use `CC0-1.0` directly?

Reply:
@PeterDaveHello lets resolve this PR, I have another updates for #5500

Reply:
Looks like CC-0 here, https://github.com/requirejs/require-jquery/blob/master/jquery-require-sample/webapp/scripts/require-jquery.js#L3 is much more older than https://github.com/requirejs/require-jquery/blob/master/LICENSE.

@sashberd please give me a link of your screen shot, I want to be careful, thanks.

Reply:
@PeterDaveHello 
Here https://github.com/requirejs/require-jquery/blob/master/jquery-require-sample/webapp/scripts/require-jquery.js#L3 the date is 22/04/2013
The commit with name "clarifying license" refers to 09/03/2016
https://github.com/requirejs/require-jquery/blob/master/LICENSE

Reply:
@PeterDaveHello Is that ref ok for you?